tests/examples/connext_dds/request_reply/quack_files/request_reply_example_test_01.ti.yml

```diff
@@ -0,0 +1,65 @@
+apps:
+  -
+    name: test_requester_py
+    product: pro
+    version: 7.4.0
```

Comment:
I though the files in the community must be compatible with the LTS. Am I wrong or this and the next should be 7.3.0?

Reply:
master branch is compatible with the LTS.
develop branch is compatible with Connext develop branch, this is what we want to test.
Currently DITTO cannot test against the weekly, but that's the idea, so we know if we break something.